public/pages/CreateTrigger/utils/helper.js

```diff
@@ -74,8 +74,10 @@ export const getTriggerContext = (executeResponse, monitor, values, triggerIndex
   if (_.isArray(trigger) && triggerIndex >= 0) trigger = trigger[triggerIndex];
 
   return {
-    periodStart: moment.utc(_.get(executeResponse, 'period_start', Date.now())).tz(getTimeZone()).format(),
-    periodEnd: moment.utc(_.get(executeResponse, 'period_end', Date.now())).tz(getTimeZone()).format(),
+    // Backend only supports, UTC timezone. 
+    // Don't use user's local timezone.
+    periodStart: moment.utc(_.get(executeResponse, 'period_start', Date.now())).format(),
```

Comment:
Could you add unit tests for the `getTriggerContext` function? The cypress tests (integration tests) are not currently configured to run with notifications plugin integration, so unit tests are the only way to confirm this is working as intended.